custom-plugins/cms131v14_diabetes_eye_exam/protocols/cms131v14_protocol.py

```diff
@@ -0,0 +1,690 @@
+)
+from canvas_sdk.value_set.v2026.intervention import (
+    HospiceCareAmbulatory,
+    PalliativeCareIntervention,
+)
+from canvas_sdk.value_set.v2026.symptom import FrailtySymptom
+from canvas_sdk.value_set.v2026.device import FrailtyDevice
+from canvas_sdk.value_set.v2026.medication import DementiaMedications
+from canvas_sdk.value_set.v2022.condition import (
+    DementiaAndMentalDegenerations,
+    Cancer,
+)
+from canvas_sdk.value_set.v2022.encounter import (
+    CareServicesInLongTermResidentialFacility,
+    NursingFacilityVisit,
+)
+from canvas_sdk.value_set.v2022.intervention import PalliativeOrHospiceCare
+from canvas_sdk.value_set.v2022.medication import ChemotherapyForAdvancedCancer
+from logger import log
+
+
+class CMS131v14DiabetesEyeExam(ClinicalQualityMeasure):
+    class Meta:
+        title = "Diabetes: Eye Exam"
+        description = (
+            "Percentage of patients 18-75 years of age with diabetes and an active diagnosis of retinopathy in any part of the measurement period who had a retinal or dilated eye exam by an eye care professional during the measurement period or diabetics with no diagnosis of retinopathy in any part of the measurement period who had a retinal or dilated eye exam by an eye care professional during the measurement period or in the 12 months prior to the measurement period"
+        )
+        version = "2025-05-08v14"
+        information = "https://ecqi.healthit.gov/ecqm/ec/2026/cms0131v14"
+        identifiers = ["CMS131v14"]
+        types = ["CQM"]
+        authors = ["National Committee for Quality Assurance"]
+        references = [
+            "American Diabetes Association. Microvascular complications and foot care. Sec. 10. In Standards of Medical Care in Diabetes 2017. Diabetes Care 2017;40(Suppl. 1):S88-S98",
+            "American Diabetes Association. (2024). Statistics About Diabetes. Retrieved from https://diabetes.org/about-diabetes/statistics/about-diabetes",
+            "Centers for Disease Control and Prevention. (2020). Common Eye Disorders and Diseases. Retrieved from https://www.cdc.gov/visionhealth/basics/ced/index.html",
+            "Centers for Disease Control and Prevention. (2022a). What is Diabetes? Retrieved from https://www.cdc.gov/diabetes/basics/diabetes.html",
+            "Centers for Disease Control and Prevention. (2022b). National Diabetes Statistics Report, 2021. US Dept of Health and Human Services. Retrieved from https://www.cdc.gov/diabetes/library/reports/reportcard.html",
+            "Centers for Disease Control and Prevention. (2024). National Diabetes Statistics Report. Retrieved from https://www.cdc.gov/diabetes/php/data-research/index.html",
+            "Parker E, Lin J, Mahoney T, et al. Economic Costs of Diabetes in the U.S. in 2022. Diabetes Care. 2024;47(1):26-43. doi:10.2337/dci23-0085"
+        ]
+
+    RESPONDS_TO = [
+        EventType.Name(EventType.CONDITION_CREATED),
+        EventType.Name(EventType.CONDITION_UPDATED),
+    ]
```

Comment:
This only listens for condition events, but could be impacted by changes in other data types. It seems like medications and interviews should impact this status, and perhaps other data types as well. 

We also need a way for patients to "age in" or "age out" of this. If a patient is eligible except they're too young, we need to make it applicable when they become old enough. Likewise, once a patient turns 76, the protocol would no longer be applicable.